spaceship/jobs/email_job.py

```diff
@@ -0,0 +1,23 @@
+from .base_job import BaseJob
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from .. import app
+
+class EmailJob(BaseJob):
+  @classmethod
+  def perform(cls, to_emails, subject, html_content, from_email='gaia@spaceshipearth.org'):
+    message = Mail(
+      from_email=from_email,
+      to_emails=to_emails,
+      subject=subject,
+      html_content=html_content)
+
+    if app.config['IN_PRODUCTION']:
+      sg = SendGridAPIClient(app.config['SENDGRID_KEY'])
+      response = sg.send(message)
+    else:
```

Comment:
Does this mean we will no longer send any emails in dev? Before, we would do so if the key was set up properly. This seems like an ok change but may be surprising for igork.

Reply:
yup, will change back to how it was, the `IN_PRODUCTION` env var is too broad an an exception handler seems better.